imports/ui/components/account/login/AccountManager.jsx

```diff
@@ -109,7 +109,7 @@ export default class AccountManager extends React.Component {
                             onLoadComplete = {this.handleHideLoading.bind(this)}
                             onSubmit={this.handleLoading.bind(this)} />} /> : null}
           {this.state.isLoading ?
-            <ModalContainer onHidden={this.handleHideLoading.bind(this)}
+            <ModalContainer disableHide={true}
```

Comment:
Noted, Option to disable/enable hide on Modal windows. Good customizability for Modals